packages/trpc/server/routers/publicViewer/noShow.handler.ts

```diff
@@ -7,15 +8,57 @@ type NoShowOptions = {
 };
 
 export const noShowHandler = async ({ input }: NoShowOptions) => {
-  const { bookingUid } = input;
-  await prisma.booking.update({
-    where: {
-      uid: bookingUid,
-    },
-    data: {
-      noShowHost: true,
-    },
-  });
+  const { bookingUid, attendees } = input;
+
+  try {
+    const attendeeEmails = attendees?.map((attendee) => attendee.email) || [];
+    if (attendees && attendeeEmails.length > 0) {
```

Comment:
Pretty much identifies if this is a guest/attendee no-show call or a host no-show call